src/dotnet/commands/dotnet-pack/PackageGenerator.cs

```diff
@@ -58,7 +59,7 @@ public bool BuildPackage(IEnumerable<ProjectContext> contexts, List<DiagnosticMe
 
             var packageOutputPath = Path.Combine(
                 ArtifactPathsCalculator.PackageOutputPath,
-                GetPackageName() + NuGet.Constants.PackageExtension);
+                GetPackageName() + NuGetConstants.PackageExtension);
```

Comment:
There is a better constant for this in NuGet.Packaging.Core which you are already using. 

``PackagingCoreConstants.NupkgExtension``